docs/admob/android.md

```diff
@@ -37,3 +37,20 @@ public class MainApplication extends Application implements ReactApplication {
   // ...
 }
 ```
+
+## Enable Legacy HTTP Library in AndroidManifest
+
+Android 9.0 removed by default all traces of the old Apache HTTP library. However, it seems that some libraries need it. You need to add this to your manifest inside the <application> element to enable legacy HTTP Library.
```

Comment:
```suggestion
Android 9.0 removed by default all traces of the old Apache HTTP library. However, it seems that some libraries need it. You'll need to add this to your manifest inside the `<application>` element of your `AndroidManifest.xml` file to enable legacy HTTP Library.
```

 - Add missing backticks to application otherwise it gets parsed as a html element and wont show on the docs
 - Tweak wording